src/bin/vip-export-sql.js

```diff
@@ -6,12 +6,19 @@ import { makeCommandTracker } from '../lib/tracker';
 
 const examples = [
 	{
-		usage: 'vip export sql @mysite.develop',
-		description: 'Export SQL file from your site and save it to the current directory',
+		usage: 'vip @example-app.develop export sql',
+		description:
+			'Download a copy of the most recent database backup for an environment as an archived SQL file to the current local directory.',
 	},
 	{
-		usage: 'vip export sql @mysite.develop --output=~/Desktop/export.sql.gz',
-		description: 'The output file can be specified with the --output flag',
+		usage: 'vip @example-app.develop export sql --output=~/Desktop/export.sql.gz',
+		description:
+			'Download the archived SQL file to a specific local directory with a custom file name.',
```

Comment:
Per our chat, let's get rid of SQL here too and reference that it is a "copy". Maybe:

> Download an archived copy of the most recent database backup for an environment to a specified file path.

Or something similar. I think it might be worth including "path" since it's not just the filename that's being defined.